thinkhazard_common/models.py

```diff
@@ -33,10 +36,14 @@ class AdminLevelType(Base):
 
     @classmethod
     def get(cls, mnemonic):
+        if mnemonic in adminleveltype_cache:
+            return adminleveltype_cache[mnemonic]
         with DBSession.no_autoflush:
-            return DBSession.query(cls) \
+            adminleveltype = DBSession.query(cls) \
                 .filter(cls.mnemonic == mnemonic) \
                 .one_or_none()
+            adminleveltype_cache[adminleveltype.mnemonic] = adminleveltype
+            return adminleveltype
```

Comment:
I've encouter some problems with session expiration in tests, see next commit, so this should not apply anymore.